src/tensor_ops.jl

```diff
@@ -1,3 +1,15 @@
+# Deprecate `*` and `'*` for tensor multiplication
```

Comment:
No keep the errors. All I meant is that these are not technically "deprecations.

> Deprecation is the discouragement of use of some feature, design or practice, typically because it has been superseded or is no longer considered safe, **without (at least for the time being) removing it from the system of which it is a part or prohibiting its use.**